subgraphs/algebra-integral/src/commons.ts

```diff
@@ -148,3 +144,20 @@ export function getOrCreatePoolToken(
 
     return poolToken;
 }
+
+export function getOrCreateTick(poolAddress: Bytes, idx: i32): Tick {
+    let id = poolAddress.concat(Bytes.fromI32(idx));
+    let tick = Tick.load(id);
+    if (tick !== null) {
+        return tick;
+    }
+
+    tick = new Tick(id);
+    tick.idx = BigInt.fromI32(idx);
```

Comment:
What's the `idx`?

Reply:
It's the tick ordinal id (i.e. index) in the tick space.